docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[searchable-snapshots]]
+== {search-snaps-cap}
+
+beta::[]
+
+{search-snaps-cap} let you reduce your operating costs by using
+<<snapshot-restore, snapshots>> for resiliency rather than maintaining
+<<scalability,replica shards>> within a cluster. When you mount an index from a
+snapshot as a {search-snap}, {es} copies the index shards to local storage
+within the cluster. This ensures that search performance is comparable to
+searching any other index, and minimizes the need to access the snapshot
+repository. Should a node fail, shards of a {search-snap} are automatically
+recovered from the snapshot repository.
+
+This can result in significant cost savings. With {search-snaps}, you may be
+able to halve your cluster size without increasing the risk of data loss or
+reducing the amount of data you can search. Because {search-snaps} rely on the
+same snapshot mechanism you use for backups, they have a minimal impact on your
+snapshot repository storage costs.
+
+[discrete]
+[[using-searchable-snapshots]]
+=== Using {search-snaps}
+
+Searching a {search-snap} is the same as searching any other index. Search
+performance is comparable to regular indices because the shard data is copied
+onto nodes in the cluster when the {search-snap} is mounted.
+
+You can control the allocation of the shards of {search-snap} indices using the
+same mechanisms as for regular indices. For example, you could use
+<<shard-allocation-filtering>> to restrict {search-snap} shards to a subset of
+your nodes.
```

Comment:
I'm ambivalent -- I put it here since we're starting off by talking about how these indices are mostly maniuplated (searched & allocated) as if they were normal indices, but I've moved it in 990707b6758d4c82243aec92cb582e34daa828ca.